src/bio2bel/sources/intact.py

```diff
@@ -0,0 +1,727 @@
+# -*- coding: utf-8 -*-
+
+"""Download and convert IntAct to BEL.
+
+Run with ``python -m bio2bel.sources.intact``
+
+`IntAct <https://www.ebi.ac.uk/intact/>`_ is a interaction database with information about interacting proteins,
+their relation, and the experiments, in which these interactions were found.
+Among the interactions that are documented in IntAct are protein modifications, associations, direct interactions,
+binding interactions and cleavage reactions.
+These interactions were grouped according to their biological interpretation and mapped to the corresponding BEL
+relation. The interactions in IntAct had a higher granularity than the interactions in BioGRID. Due to the default BEL
+namespace of protein modifications :data:`pybel.language.pmod_namespace`, the post-translational protein modification
+can be identified very accurately. For example, the glycosylation of a protein can be described in BEL by
+:code:`pybel.dsl.ProteinModification('Glyco').  Although many protein modifications had corresponding terms in BEL,
```

Comment:
you don't believe this needs a backtick at the end? I told you two times now :/

Reply:
also fix the double space

Reply:
This is what I've been telling you about. I fixed it (multiple times), committed, pushed and then its gone again. I really can't explain it

Reply:
solution to the problem is to read your diffs before pushing